fenix/app/src/main/java/org/mozilla/fenix/addons/AddonsManagementFragment.kt

```diff
@@ -192,10 +192,9 @@ class AddonsManagementFragment : Fragment(R.layout.fragment_add_ons_management)
         binding?.addonProgressOverlay?.cancelButton?.setOnClickListener {
             lifecycleScope.launch(Dispatchers.Main) {
                 val safeBinding = binding
-                // Hide the installation progress overlay once cancellation is successful.
-                if (installOperation.cancel().await()) {
```

Comment:
We need to remove this, as when we are trying to cancel an install request, we can have two signal one, `true` telling use the the cancellation was successful and another `false` telling use that cancellation wasn't possible, either way we have to hide the overlay.